src/config/env.ts

```diff
@@ -16,8 +16,10 @@ export const SHMO_INVITE_KEY: string | undefined = process.env.SHMO_INVITE_KEY;
 export const ET_CLIENT_NAME = process.env.CLIENT_NAME || 'atb-bff';
 
 export const PROJECT_ID = process.env.PROJECT_ID;
+export const API_BASE_URL: string | undefined = process.env.API_BASE_URL;
```

Comment:
Why use this instead of `getServiceUrl()`?

https://github.com/AtB-AS/atb-bff/blob/5ec2e81269e2a4a36ca15d5d0e6c24952b1d39b5/src/config/env.ts#L68-L74

For sales service, the service key is `SALES`.